stock_picking_auto_create_lot_qty/models/product_template.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    batch_uom_id = fields.Many2one(
+        string="Uom",
+        help="The unit of measure used when creating batches.",
+        comodel_name="uom.uom",
```

Comment:
why do we need this?

Reply:
This is one of the conditions of the customer, it is necessary to create a batch